spec/linter-rubocop-spec.js

```diff
@@ -0,0 +1,126 @@
+'use babel';
+
+import * as path from 'path';
+
+const lint = require(path.join('..', 'lib', 'index')).provideLinter().lint;
+
+const badPath = path.join(__dirname, 'fixtures', 'bad.rb');
+const emptyPath = path.join(__dirname, 'fixtures', 'empty.rb');
+const goodPath = path.join(__dirname, 'fixtures', 'good.rb');
+const invalidPath = path.join(__dirname, 'fixtures', 'invalid.rb');
+
+describe('The RuboCop provider for Linter', () => {
+  beforeEach(() => {
+    atom.workspace.destroyActivePaneItem();
+    waitsForPromise(() => {
+      atom.packages.activatePackage('linter-rubocop');
+      return atom.packages.activatePackage('language-ruby').then(() =>
+        atom.workspace.open(goodPath)
+      );
+    });
+  });
+
+  it('should be in the packages list', () =>
+    expect(atom.packages.isPackageLoaded('linter-rubocop')).toEqual(true)
+  );
+
+  it('should be an active package', () =>
+    expect(atom.packages.isPackageActive('linter-rubocop')).toEqual(true)
+  );
+
+  describe('shows errors in a file with errors', () => {
+    let editor = null;
+
+    beforeEach(() => {
+      waitsForPromise(() =>
+        atom.workspace.open(badPath).then(openEditor => { editor = openEditor; })
+      );
+    });
+
+    it('finds at least one message', () => {
+      waitsForPromise(() =>
+        lint(editor).then(messages =>
+          expect(messages.length).toBeGreaterThan(0)
+        )
+      );
+    });
+
+    it('verifies the first message', () => {
+      const msgText = 'unterminated string meets end of file\n(Using Ruby 2.2 parser; ' +
+        'configure using `TargetRubyVersion` parameter, under `AllCops`) (Syntax)';
+
+      waitsForPromise(() =>
+        lint(editor).then(messages => {
+          expect(messages[0].type).toBeDefined();
```

Comment:
Duplicate test essentially, it will continue on and complain about it not existing if this _were_ to fail on the next test so there isn't really a point to have this here.